html-skeleton/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE>
+
+<html>
+
+<!-- non visible content, also known as metadata -->
+<!-- white area in browser is called viewport -->
+  <head>
+    <!-- title does contain something visual but does it on the tab -->
+    <title>HTML Skeletons</title>
+  <body>
```

Comment:
You are missing the closing tag for the `head` element

Reply:
Oh man..
Thanks, will fix!